modules/openapi-generator/src/test/resources/3_0/python/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -2272,4 +2272,16 @@ components:
           properties:
             skill:
               type: string
-
+    AdditionalPropertieObject:
+      type: object
+      properties:
+        name:
+          type: string
+      additionalProperties:
+        type: object
+    AdditionalPropertieAnyType:
+      type: object
+      properties:
+        name:
+          type: string
+      additionalProperties: {}
```

Comment:
I had one more case, where there is no type specified, but there is a description field:
```yaml
additionalProperties:
    description: This is what the additional property is
```

Reply:
that's the same as any type: https://swagger.io/docs/specification/data-models/data-types/#any

but I can definitely add another test to make sure there's no surprise.

Reply:
Looks good. Nice to be safe!

Reply:
added